README.md

```diff
@@ -22,6 +22,7 @@ runners are supported:
 |       Language | Test Runners                                                                                                       | Identifiers                                                                                                                                  |
 | -------------: | :----------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------- |
 |         **C#** | .NET                                                                                                               | `xunit`, `dotnettest`                                                                                                                        |
+|         **C++** | ctest                                                                                                               | `ctest`, `dotnettest`                                                                                                                        |
```

Comment:
Should `dotnettest` be part of this or is this a typo?

Reply:
Typo, thanks for the catch! I have updated the README with the correct info